XyzWalker/Assets/Src/Scripts/Boy.cs

```diff
@@ -0,0 +1,76 @@
+// Copyright 2021-2022 The SeedV Lab.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System.Collections;
+using UnityEngine;
+
+public class Boy : MonoBehaviour {
+  private const string _walkTrigger = "Walk";
+  private const string _standTrigger = "Stand";
+  private const string _jumpTrigger = "Jump";
+  private float _defaultX = -45f;
```

Comment:
Should _defaultX, _defaultZ, _defaultAngleY be const variables, and never be changed?

Reply:
Done.